README.md

```diff
@@ -2,3 +2,33 @@
 
 This repository contains the scripts and other materials needed to execute the
 workshop for SIG Automation at the 2022 ONE Summit in Seattle.
+
+## Setup Details
+
+### Virtual Machine Details
+
+| <!-- -->         | <!-- -->     |
+|------------------|--------------|
+| Operating System | Ubuntu 22.04 |
+| Architecture     | x86_64       |
+| Cores            | 8            |
+| RAM              | 32 GB        |
+| Disk             | TBD          |
+
+
+
+### Pre-Installed Software
+- docker
+- git
+-
+
+### KIND Clusters
+KIND Version -> TBD
```

Comment:
IMHO the only reason to not use the latest stable KinD version is when you're trying to use older K8s versions.
```suggestion
KIND Version -> 0.17.0
```